concordium-contracts-common/src/lib.rs

```diff
@@ -97,7 +107,7 @@ mod types;
 #[cfg(feature = "smart-contract")]
 pub use concordium_contracts_common_derive::*;
 #[cfg(not(feature = "smart-contract"))]
-pub use concordium_contracts_common_derive::{Deserial, Serial, Serialize};
+pub use concordium_contracts_common_derive::{Deserial, SchemaType, Serial, Serialize};
```

Comment:
Why this? Typicallly schematype should only be needed in smart contracts, at least it should only be derived there.

Reply:
Some of the `PublicKey, CredentialPublicKeys, ...` derive it but I moved the import into the `types.rs` file since it is used there.